ui/src/keyboardMappings.ts

```diff
@@ -333,3 +247,44 @@ export const keyDisplayMap: Record<string, string> = {
   "(PrintScreen)": "sys rq", "(Pause)": "break",
   SystemRequest: "sys rq", Break: "break"
 };
+
```

Comment:
These will be used when we start _correctly_ reporting the language in the HID report